File: Sorting Algo/linearSearch.java
public class LinearSearch {

    public static int linearSearch(int[] arr, int element) {     //parameter are taken
        for (int i = 0; i < arr.length; i++) {                  //loop iterates through array
            if (arr[i] == element) {                            //compares each element with element
               return i;
            }
        }
        return -1;                                              //returns -1 if not found and index if found         
    }

    public static void main(String[] args) {
        int[] numbers = {8,14,65,23,19};
        int element = 14;
        int result = linearSearch(numbers, element);            //calls linear search function to search of the element
        
        if (result != -1) {
            System.out.println("Element found at index: " + result);
        } else {
            System.out.println("Element not found.");
        }
    }
}

File: Searching Algo/linear_search.py

def linear_search(arr,target):
  for i in range (len(arr)):
    if arr[i]==target:
      return i;
    return -1;
  
#testing
arr=[2,4,6,8,10]
target=10

result=linear_search(arr,target)
  
  
if result != -1:
    print(f"Element found at index {result}")
else:
    print("Element not found in the array")
    
    
  